proto/meta.proto

```diff
@@ -83,7 +83,9 @@ message TableFragments {
     repeated stream_plan.StreamActor actors = 4;
     // Vnode mapping (which should be set in upstream dispatcher) of the fragment.
     // This field is always set to `Some`. For singleton, the parallel unit for all vnodes will be the same.
```

Comment:
nit
```suggestion
```

Reply:
Yeah, I think it's definitely good to have some comments mentioning what it was and why it was deprecated (replaced by what), but leaving the original comment might be confusing.